multiversx_sdk/core/transactions_outcome_parsers/transaction_events_parser_test.py

```diff
@@ -167,3 +169,35 @@ def test_parse_event_with_multi_values():
         b=["test", 44],
         c=42
     )
+
+
+@pytest.mark.networkInteraction
+def test_multisig_start_perform_action():
+    from multiversx_sdk import (ApiNetworkProvider, TransactionEventsParser,
+                                TransactionsConverter,
+                                find_events_by_first_topic)
+    from multiversx_sdk.abi import Abi
+
+    api = ApiNetworkProvider("https://testnet-api.multiversx.com")
+    converter = TransactionsConverter()
+
+    transaction_on_network = api.get_transaction("6f006c99e45525c94629db2442d9ca27ff088ad113a09f0a3a3e24bcc164945a")
+    transaction_outcome = converter.transaction_on_network_to_outcome(transaction_on_network)
+
+    abi = Abi.load(Path(testdata / "multisig-full.abi.json"))
```

Comment:
`testdata` is already a `Path`, can be directly `testdata / "multisig-full.abi.json"`.

Reply:
indeed, done